app/subscriber/src/features/my-folders/MyFolders.tsx

```diff
@@ -34,15 +34,15 @@ export const MyFolders = () => {
     }).then((data) => {
       toast.success(`${data.name} created successfully`);
       setNewFolderName('');
-      setMyFolders([...myFolders, data]);
+      setMyFolders([data]);
     });
   };
 
   const handleSave = () => {
     if (!!active) {
-      updateFolder(active).then(() => {
+      updateFolder(active).then((data) => {
         toast.success(`${active.name} updated successfully`);
-        setMyFolders([...myFolders.filter((folder) => folder.id !== active.id), active]);
+        setMyFolders([data]);
```

Comment:
Appears like this might be problematic.  This would result in loosing any other folders currently in state.  Is this what you want?